client/components/Reviews/components/FormElement.jsx

```diff
@@ -1,26 +1,43 @@
+/* eslint-disable react/jsx-props-no-spreading */
 /* eslint-disable jsx-a11y/label-has-for */
+/* eslint-disable react/require-default-props */
```

Comment:
I noticed you use these eslint-disable comments in your files! Are any of these settings we can add to our eslint config? Or are these only in place temporarily? No changes needed, just curious if we can automate this for you!

Reply:
I think in some cases I might not always agree with the rule for my use case. I don't think I will need these rules to be disabled though!